content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -230,30 +226,75 @@ clusters, zones, or regions.
+- One control plane per cluster (**highest availability**, **highest isolation**)
+
+## Mesh models
+
+Istio supports having all of your services in a
+{{< gloss "service mesh" >}}single mesh{{< /gloss >}}, or federating multiple meshes
+together, which is also known as {{< gloss >}}multi-mesh{{< /gloss >}}.
+
+### Single mesh
+
+The simplest Istio deployment is a single mesh. A single mesh can span
+[one or more clusters](#cluster-models) and [one or more networks](#network-models).
+All workloads in a mesh share a common certificate authority. You can choose to use a single shared
+  trust domain or a separate trust domain for each cluster.
+If you use a single shared trust domain, a service account in a given namespace is globally unique - i.e. if
+two clusters `cluster-1` and `cluster-2` have the same service account `foo` in namespace `ns`, they will be
+treated the same for purposes of security. With a trust domain for each cluster, there are no
+restrictions on the uniqueness of service accounts, and namespaces. You can apply enforcement
+policies that allow or deny all traffic from one cluster (irrespective of the namespace/service) to another.
 
-- Configuration isolation: You can make configuration changes in one cluster,
-  zone, or region without impacting others.
+### Multiple meshes
+
+Multiple mesh deployments result from {{< gloss >}}mesh federation{{< /gloss >}}, with the following capabilities:
+
+- Organizational boundaries: lines of business.
+- Potential for cross vendor meshes to communicate with each
+  other if the trust bundles are setup properly.
 
-You can improve control plane availability through fail over. When a control
-plane instance becomes unavailable, workload instances can connect to
-another available control plane instance. Fail over can happen across clusters,
-zones, or regions.
+You can enable inter-mesh communication with {{< gloss >}}mesh federation{{</gloss >}}.
+When federating, each mesh can expose a set of services and identities, which all participating meshes can recognize.
 
 {{< image width="50%"
-    link="failover.svg"
-    alt="A service mesh after a control plane instance fails"
-    title="Control plane fail over"
-    caption="A service mesh after a control plane instance fails"
+    link="multi-mesh.svg"
+    alt="Multiple service meshes"
+    title="Multi-mesh"
+    caption="Multiple service meshes"
     >}}
 
-The following list ranks control plane deployment examples by availability:
-
-- One cluster per region (**lowest availability**)
-- Multiple clusters per region
-- One cluster per zone
-- Multiple clusters per zone
-- Each cluster (**highest availability**)
+When federating two meshes that do not share the same
+{{< gloss >}}trust domain{{< /gloss >}}, you must
+{{< gloss "mesh federation">}}federate{{< /gloss >}}
+{{< gloss >}}identity{{< /gloss >}} and **trust bundles** between them. See the
+section on [Multiple Trust Domains](#trust-between-meshes) for an overview.
```

Comment:
Here we are referring to multiple trust domains to a link that is trust between meshes.  but earlier the doc says you could have multiple trust domains for a single mesh (each cluster has its own trust domain).

Reply:
Why does trust domain (an opaque string) have to be tied to the CA or mesh? Intuitive is very subjective. To me, it is unintuitive to run single Istio for many clusters or allow pods in one cluster to access internal k8s services of another cluster :). Yet, we have many formulations of this (single/independent/central control planes). The trust domain falls under the same umbrella. You can use a single global trust domain for entire mesh (like central istiod) or many trust domains (one per cluster). Its just essentially build your own adventure, like the rest of this doc.

Reply:
good catch.. will remove